.github/workflows/vercel-deploy.yml

```
@@ -0,0 +1,155 @@
+        
+    - name: 📊 Comment PR with deployment info
+      if: github.event_name == 'pull_request'
+      uses: actions/github-script@v7
+      with:
+        script: |
+          github.rest.issues.createComment({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: `🎭 **Bot WhatsApp RP Déployé!**
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)